### ISSUE TITLE
Color in UI: DL and Carb are mirrored page designs

### ISSUE THREAD

**Issue by mjabbink** [carbon-design-system/design-language-website#167]
the issues we would create for this page still apply because they are the same for Carbon.

For mobile there are specific mobile friendly images. The “Digital accessibility” section for example should be like current DL

https://www.ibm.com/design/language/elements/color-ui

Expected:

![33766D9F-5FDF-4096-84A7-B8EFD268C3DB](https://user-images.githubusercontent.com/32881239/62831491-e014d780-bc3d-11e9-8112-8936bdec6a02.jpeg)

Staging:

![4D9FF76E-3E16-4E4F-BA1B-5C34478AF0B4](https://user-images.githubusercontent.com/32881239/62831494-f3c03e00-bc3d-11e9-9c58-26e88483d86c.jpeg)

**Comment by mjabbink** [comment]
There are some images in “Interactive states) that are showing Times New Roman and a blurry hover hand icon. @sadek should have the source because broken in live IDL too

**Comment by mjabbink** [comment]
There is a section missing called “Commonly used colors” and not sure if you all wanted to edit this out? @jeannie

**Comment by emyarod** [comment]
@abbeyhrt the image changes can probably be included in your ongoing mobile images work right?

**Comment by alisonjoseph** [comment]
I thought we were removing UI pages from DL? (we can move issues over to Carbon)
